lib/hexpm/web/views/layout_view.ex

```diff
@@ -42,9 +42,17 @@ defmodule Hexpm.Web.LayoutView do
       tag(:meta, property: "og:title", content: Map.get(assigns, :title)),
       tag(:meta, property: "og:type", content: "website"),
       tag(:meta, property: "og:url", content: Map.get(assigns, :canonical_url)),
-      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/hex.png")),
+      tag(:meta, property: "og:image", content: static_url(Hexpm.Web.Endpoint, "/images/open-graph.png")),
```

Comment:
There's a guideline for og image, and github doesn't follow that. It actually  gets the gravatar of the user/organization.

Instead I followed the doc for twitter and facebook for that. Also checked npm.org, they use the same approach.